app/components/Navigation.js

```diff
@@ -2,9 +2,13 @@ import React, { Component } from 'react';
 import { Nav, Navbar, Button } from 'react-bootstrap';
```

Comment:
watch linting. i think the original component you were working from was inconsistent